known_issue: Add NCSDK-13858 (#7634)
nrfconnect/sdk-nrf: doc/nrf/known_issues.rst

nRF Desktop
===========

.. rst-class:: v1-9-1 v1-9-0 v1-8-0

> NCSDK-13858: Possible crash on BLE advertising start when using SW Split Link Layer

Reviewer: ```suggestion
NCSDK-13858: Possible crash at the start of Bluetooth LE advertising when using SW Split Link Layer
```

>   nRF Desktop peripheral may crash on advertising start when using SW Split Link Layer (:kconfig:option:`CONFIG_BT_LL_SW_SPLIT`).

Reviewer: ```suggestion
  The nRF Desktop peripheral can crash at the start of the advertising when using SW Split Link Layer (:kconfig:option:`CONFIG_BT_LL_SW_SPLIT`).
```

>   The crash is caused by issue of the Bluetooth controller.

Reviewer: ```suggestion
  The crash is caused by an issue of the Bluetooth Controller.
```

>   The used size of resolving list filter is invalid which leads to accessing memory out of array bonds.

Reviewer: ```suggestion
  The size of the resolving list filter is invalid, which causes accessing memory areas that are located out of array.
```

>   **Workaround:** Manually cherry-pick and apply commit with fix to sdk-zephyr (commit hash: ``15ebdfafe2b2932533aa8d71afd49d4b03d27ce4``).

Reviewer: ```suggestion
  **Workaround:** Manually cherry-pick and apply commit with fix to ``sdk-zephyr`` (commit hash: ``15ebdfafe2b2932533aa8d71afd49d4b03d27ce4``).
```
